stdlib/tkinter/__init__.pyi

```diff
@@ -1112,17 +1115,137 @@ class Canvas(Widget, XView, YView):
     def canvasx(self, screenx, gridspacing: Optional[Any] = ...): ...
     def canvasy(self, screeny, gridspacing: Optional[Any] = ...): ...
     def coords(self, *args): ...
-    def create_arc(self, *args, **kw): ...
-    def create_bitmap(self, *args, **kw): ...
-    def create_image(self, *args, **kw): ...
-    def create_line(self, *args, **kw): ...
-    def create_oval(self, *args, **kw): ...
-    def create_polygon(self, *args, **kw): ...
-    def create_rectangle(self, *args, **kw): ...
-    def create_text(self, *args, **kw): ...
-    def create_window(self, *args, **kw): ...
+    def create_arc(self, *args, **kw) -> _ObjectId: ...
+    def create_bitmap(self, *args, **kw) -> _ObjectId: ...
+    def create_image(self, *args, **kw) -> _ObjectId: ...
+    def create_line(self, x0: int, y0: int, x1: int, y1: int, /,
+        activedash: _Color = ...,
+        activefill: _Color = ...,
+        activestipple: str = ...,
+        activewidth: Union[int, str] = ...,
+        arrow: Literal["first", "last", "both"] = ...,
+        arrowshape: Tuple[int, int, int] = ...,
```

Comment:
Nit: Built-in generics are now mostly supported for typeshed.

```suggestion
        arrowshape: tuple[int, int, int] = ...,
```

(And a few more `Tuple` instances below.)

Reply:
There seems to be a problem with `tuple`, as just changing `Tuple` to `tuple` causes mypy primer failures unrelated to canvas. Maybe this is the same tuple problems as in #4820 ?